src/test/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifierIntegrationTest.java

```diff
@@ -95,12 +97,22 @@ private AbstractBuild mockRun() {
 
     private Job mockJob() {
         Job job = mock(Job.class);
+        ItemGroup itemGroup = mockItemGroup();
 
         when(job.getDisplayName()).thenReturn(JOB_NAME);
+        when(job.getParent()).thenReturn(itemGroup);
```

Comment:
is already mocked in https://github.com/jenkinsci/office-365-connector-plugin/pull/53/files#diff-c3445d01c41463f6a00a6ba618774b8fR84 so which one is correct?

Reply:
the default itemGroup and then there's Multi Branch Project item group.

Reply:
Also job's parent cannot be null otherwise we get an illegal state exception :(

Reply:
Removed was not needed for this test. Was mocking about earlier 😜